### DISCUSSION TITLE
Mcrypt PHP extension required

### DISCUSSION THREAD

**Discussion by sandimvlyadi** [shivammathur/setup-php#622] [Q&A / Help] [answered]
Hi, I use apple silicon and install php5.6 but mcrypt extension is missing. How to fix it? Thanks

**Comment by shivammathur** [comment, marked as answer]
Use [shivammathur/extensions](https://github.com/shivammathur/homebrew-extensions) tap

```zsh
brew tap shivammathur/extensions
brew install shivammathur/extensions/mcrypt@5.6
```
